docs/overlay.md

```diff
@@ -6,7 +6,12 @@
 |---|---|---|---|
 | `image` | `ImageSource` | A custom image to be used as the overlay. Only required local image resources and uri (as for images located in the net) are allowed to be used.
 | `bounds` | `Array<LatLng>` |  | The coordinates for the image (left-top corner, right-bottom corner). ie.```[[lat, long], [lat, long]]```
+| `bearing` | `Float` | `0` | `Google Maps API only` The bearing in degrees clockwise from north. Values outside the range [0, 360) will be normalized.
 | `tappable` | `Bool` | `false` | `Android only` Boolean to allow an overlay to be tappable and use the onPress function.
+| `location` | `[Float, Float]` |  | `Android only` Location of the image overlay.
```

Comment:
This is a LatLng according to the code

Reply:
[LatLng](https://github.com/react-native-community/react-native-maps/blob/bac105efbeee43c335ae81082b054ca43447fbea/index.d.ts#L18) is is an object with two fields :
```
export interface LatLng {
   latitude: number;
   longitude: number;
}
```
 
The `location` prop that we are passing in javascript requires a [Coordinate](https://github.com/react-native-community/react-native-maps/blob/bac105efbeee43c335ae81082b054ca43447fbea/index.d.ts#L478) object:

`type Coordinate = [number, number];`

which is pretty much an array of two numbers.

So we are actually passing a `Coordinate`, not a `LatLng`.